package.json

```diff
@@ -26,7 +26,9 @@
     "build": "next build",
     "start": "rm -r .next && next build && next start",
     "lint": "eslint 'src/**/*.{js,ts,tsx}'",
-    "prettier:format": "yarn prettier './**/*.{js,json,md,ts,tsx,yml}' --write && yarn run lint --fix"
+    "prettier:format": "yarn prettier './**/*.{js,json,md,ts,tsx,yml}' --write && yarn run lint --fix",
+    "e2e:open": "env-cmd -f .e2e.env yarn synpress open --configFile synpress.config.ts",
+    "e2e:run": "env-cmd -f .e2e.env yarn synpress run --configFile synpress.config.ts"
```

Comment:
Should we also have a `test:e2e` command here, for consistency with token bridge?

Reply:
I'm not sure it make sense. We have it in the bridge if we call it from the root